bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/editor/js/summernote_init.js

```diff
@@ -0,0 +1,34 @@
+$(function() {
+	var summernoteconfig = ${summernoteconfig};
+	var summernoteconfigdefault = {
+		onImageUpload : function(files) {
+			$(files).each(function(){
+				var file = this;
+				var data = new FormData();
```

Comment:
According to http://caniuse.com/#search=FormData only IE Edge supports FormData without issues.
We may need to use iframes for the uploads instead.